frontend/public/components/_quota.scss

```diff
@@ -5,3 +5,29 @@
 .co-resource-quota-empty {
   font-size: 15px;
 }
+
+.co-resource-quota-chart-row {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  padding-left: 200px;
+  padding-right: 200px;
+  margin-bottom: 20px;
+  .co-resource-quota-chart {
+    min-width: 150px;
+  }
+}
+
+.co-resource-quota-scope__label {
+  font-weight: 500;
+}
+.co-resource-quota-scope__description {
+  font-size: 10px;
+  font-weight: 300;
+}
+
+.co-resource-quota-infotip {
```

Comment:
This isn't the first infotip we have in the console. We should make sure we're styling them all consistently instead of having a special style for quota.

Reply:
Can u pls send me link to where else in the console?  Seaching for `infotip` and `popup` aren't finding anything in .scss files -thanks

Reply:
Looks like we used `FieldLevelHelp` in a similar context (by the header in the env var editor). We should try to be consistent between the two.

https://github.com/openshift/console/blob/master/frontend/public/components/environment.jsx#L439-L440

Reply:
Now using `FieldLevelHelp` for infotip

Reply:
@spadgett Isn't there an info tip on the environment variables section of deployments?